nh_clinical/tests/user_allocation/helpers.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+def create_test_data(cls, cr, uid, iterations=3):
```

Comment:
Although it would've been good to use the demo data I can't actually move the tests into the demo data module because then they would no longer be run with `nh_clinical` module. It's important that this happens as they are testing development code in this module.

We discussed a plan to make all modules responsible for adding their own test data and that will allow us to resolve this issue in the future.